src/stan/services/pathfinder/multi.hpp

```diff
@@ -104,138 +107,200 @@ inline int pathfinder_lbfgs_multi(
     std::vector<SingleDiagnosticWriter>& single_path_diagnostic_writer,
     ParamWriter& parameter_writer, DiagnosticWriter& diagnostic_writer,
     bool calculate_lp = true, bool psis_resample = true) {
+  using stan::services::pathfinder::internal::write_times;
   const auto start_pathfinders_time = std::chrono::steady_clock::now();
   std::vector<std::string> param_names;
   param_names.push_back("lp_approx__");
   param_names.push_back("lp__");
+  param_names.push_back("pathfinder__");
```

Comment:
I think this should be `path__` or `path_num__`

Also, I think it is missing from some of the outputs depending on which branch you end up following here

Reply:
I'm fine with that. For single I missed a branch